packages/angular/cli/models/architect-command.ts

```diff
@@ -135,45 +135,47 @@ export abstract class ArchitectCommand<
     return await this.runArchitectTarget(options);
   }
 
+  protected async runSingleTarget(targetSpec: TargetSpecifier, options: string[]) {
+    // We need to build the builderSpec twice because architect does not understand
+    // overrides separately (getting the configuration builds the whole project, including
+    // overrides).
+    const builderConf = this._architect.getBuilderConfiguration(targetSpec);
+    const builderDesc = await this._architect.getBuilderDescription(builderConf).toPromise();
+    const targetOptionArray = await parseJsonSchemaToOptions(this._registry, builderDesc.schema);
+    const overrides = parseArguments(options, targetOptionArray);
+
+    if (overrides['--']) {
+      (overrides['--'] || []).forEach(additional => {
```

Comment:
`(overrides['--'] || [])` should be redundant here  since you are already checking if it's not falsely on line 147

Reply:
Tell that to typescript :(